docs/ondat-portal/installation-portal.md

```diff
@@ -1,43 +1,38 @@
 ---
-title: "Ondat Portal Installation Guide"
+title: "Ondat SaaS Platform Installation Guide"
 linkTitle: "Ondat Portal Installation Guide"
-weight: 10
+weight: 1
 ---
 
-> ⚠️ Warning: As a prerequisite, you need to enable port 8883 for egress in your ACLs if a VPC is used.
+## Overview
 
-# Creating Credentials for Your Cluster
+This guide will demonstrate how to install the [Ondat SaaS Platform](https://portal.ondat.io/).
+
+## Prerequisite
+
+> ⚠️ Make sure the kubectl storageos plugin is installed. Follow the [install guide for kubectl storageos](https://docs.ondat.io/docs/reference/kubectl-plugin/).
+
+> ⚠️ Make sure to add an [Ondat licence](/docs/operations/licensing/) after installing.
+
+> ⚠️ You must enable port 8883 for egress in your ACLs if a VPC is used.
+
+## Procedure
+
+### Step 1: Creating Credentials for Your Cluster
 
 1. Open [Ondat Portal](https://portal.ondat.io/dashboard).
-2. Log into your account using your account's credentials.
-3. On the lower left-hand side of the screen, open the __Organization__ tab.
-4. Open the __API Tokens__ tab and select __Create API Token__.
-5. Enter a name for the API token. Note that, if there is another API Token with the same description that token will be replaced by this one.
-6. Copy all information on the page, this will be the only time it will be visible. Make note of the API secret.
-
-# Installing Ondat on Your Cluster Using the Ondat Portal Manager
-
-Execute this command if you are configuring a cluster without having installed Ondat beforehand. You can use the credentials you have just created to execute the command below:
-
-```bash
-kubectl storageos install 
-        --include-etcd=true 
-        --stos-version=develop 
-        --enable-portal-manager 
-        --portal-client-id=<clientid> 
-        --portal-secret=<secret> 
-        --portal-api-url=<api-url> 
-        --portal-tenant-id=<tenantId>
-```
-
-# Installing Ondat Portal Manager
-
-Execute this command if you are configuring a cluster in which Ondat has already been installed.
-
-```bash
-kubectl storageos install-portal 
-        --portal-client-id=<clientid> 
-        --portal-secret=<secret> 
-        --portal-api-url=<api-url> 
-        --portal-tenant-id=<tenantId>
-```
+2. Log into your account using your credentials.
+3. On the left-hand side of the screen, open the __Cluster__ tab.
```

Comment:
```suggestion
3. In the main navigation, open the __Cluster__ tab.
```

This is pretty subjective, so feel free to ignore. 

For website documentation I try to avoid positional terms, as the layout can change depending on the screen size. e.g. a left hand nav bar might collapse into a 'burger' button on mobile screen sizes. 

The portal doesn't actually have any mobile styles yet. So this is kind of a moot point. But I think it's a good habit to work to 🤷

Reply:
that is a great idea, thank you !